### DISCUSSION TITLE
GO CD agent INSTALLION ERROR

### DISCUSSION THREAD

**Discussion by vikramdevopsb79** [learndevopsonline/learndevopsonline#1908] [Q&A] [answered]
![Screenshot (424)](https://github.com/learndevopsonline/learndevopsonline/assets/169573156/fdc875a9-4909-4299-922d-20e6bba61d52)
![Screenshot (425)](https://github.com/learndevopsonline/learndevopsonline/assets/169573156/f21f82a5-1ac3-41ad-8193-a768588e878a)
actually  for root user i used curl command to extract  it's wrong step so i redone extract files  for go agent with go user , now while starting service getting above error .  i got below mentioned  error also but i ignored 

 rm -f  /tmp/go-agent-23.5.0-18179.zip
rm: cannot remove '/tmp/go-agent-23.5.0-18179.zip': Operation not permitted

**Comment by chaithanya1812** [comment]
Hi @vikramdevopsb79 ,
I am unable to login, can you start the server?

**Reply by vikramdevopsb79** [reply]
> Hi @vikramdevopsb79 , I am unable to login, can you start the server?

let me know the preferred time to start ...?

**Comment by chaithanya1812** [comment]
Hi @vikramdevopsb79,
Now you can start your servers, and paste both IP-address of go-server & go-agent in the chat.

**Reply by vikramdevopsb79** [reply]
> Hi @vikramdevopsb79, Now you can start your servers, and paste both IP-address of go-server & go-agent in the chat.

started :
ansible Public Ip : 54.81.84.41
gocd-server public Ip : 184.73.64.252

**Comment by chaithanya1812** [comment, marked as answer]
Hi @vikramdevopsb79,
## Issue:- while editing a file you unexpectedly done small mistake
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/97336fa8-701e-4229-9960-ce464b0a93c5)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/b03a667d-9c12-4236-bba2-78704be06b0d)
